config/configmaps/files/config.yaml

```diff
@@ -28,4 +28,4 @@ DSPO:
       ConnectionTimeout: $(DSPO_HEALTHCHECK_DATABASE_CONNECTIONTIMEOUT)
     ObjectStore:
       ConnectionTimeout: $(DSPO_HEALTHCHECK_OBJECTSTORE_CONNECTIONTIMEOUT)
-  RequeueTime: $(DSPO_REQUEUE_TIME)
+  RequeueTime: $(REQUEUE_TIME)
```

Comment:
Shouldn't it be `DSPO_REQUEUE_TIME` itself? The field path in the kustomize yaml points to `DSPO_REQUEUE_TIME`: https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/base/kustomization.yaml#L135

Reply:
Actually nvm, let'd just stick to `REQUEUE_TIME` everywhere. 

I think this would've worked if the variable in `params.env` were `REQUEUE_TIME` instead of `DSPO_REQUEUE_TIME`: https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/base/params.env

Could you update it to be `REQUEUE_TIME` here and in the kustomization.yaml too?

Reply:
so i checked the full codebase.... i wonder if we should converge on `DSPO_REQUEUETIME` such that it would theoretically also be picked up by viper?  it wouldn't _really_ matter since the value would be reflected in both envvars and config, but it would remove some confusion

Reply:
Sounds good, let's converge on `DSPO_REQUEUETIME`